File: year 1/python/swaping.py
print("This program swap two variable witout using third variable \n\n")

num1 = float(input("Enter a number : "))
num2 = float(input("Enter another number : "))

print("The two number: number 1 = {0} and number 2 = {1}".format(num1, num2))

# num2 = num1 + num2
# num1 = num2 - num1
# num2 = num2 - num1

num1, num2 = num2, num1

print("After Swapping:")
print("The two number: number 1 = {0} and number 2 = {1}".format(num1, num2))

File: year 2/cpp/OOPs/experiment1/swaping.cpp
#include <bits/stdc++.h>
using namespace std;

void swap(int* n1, int* n2);
int main() {

    int num1, num2;

    cout << "Enter first number: ";
    cin >> num1;
    cout << "Enter second number: ";
    cin >> num2;

    swap(&num1, &num2);

    return 0;
}


void swap(int* n1, int* n2) {
    int temp = *n1;
    *n1 = *n2;
    *n2 = temp;

    cout << "After swapping, firstNumber = " << *n1 << "\n";
    cout << "After swapping, secondNumber = " << *n2;
}